README.md

```diff
@@ -225,6 +225,7 @@ Table of Contents
   * http://azure.microsoft.com/ - MS Azure gives $200 worth of free usage for a trial
   * http://hpcloud.com/ - $300 credit over 90 days.
   * https://appharbor.com/ - A .Net PaaS that provides 1 free worker
+  * https://shellycloud.com/ - Platform for hosting Ruby and Ruby on Rails apps. Shelly Cloud gives €20 free credit
```

Comment:
hmm, that's not even 30 days of one tiny instance, dont think that fits the bill for this list.  There are other free tiers listed here but they tend to be a bit more substantial.